src/Layer/OGC3DTilesLayer.js

```diff
@@ -10,6 +10,50 @@ import ReferLayerProperties from 'Layer/ReferencingLayerProperties';
 // Internal instance of GLTFLoader, passed to 3d-tiles-renderer-js to support GLTF 1.0 and 2.0
 // Temporary exported to be used in deprecated B3dmParser
 export const itownsGLTFLoader = new iGLTFLoader();
+// TODO: waiting for https://github.com/NASA-AMMOS/3DTilesRendererJS/pull/626 to be merged to enable this extension
+// itownsGLTFLoader.register(() => new GLTFMeshFeaturesExtension());
+itownsGLTFLoader.register(() => new GLTFStructuralMetadataExtension());
+itownsGLTFLoader.register(() => new GLTFCesiumRTCExtension());
+
+// Instantiated by the first tileset. Used to share cache and download and parse queues between tilesets
+let lruCache = null;
+let downloadQueue = null;
+let parseQueue = null;
+
+export const OGC3DTILES_LAYER_EVENTS = {
+    /**
+     * Fired when a new root or child tile set is loaded
+     * @event OGC3DTilesLayer#load-tile-set
+     * @type {Object}
+     * @property {Object} tileset - the tileset json parsed in an Object
+     * @property {String} url - tileset url
+     */
+    LOAD_TILE_SET: 'load-tile-set',
+    /**
+     * Fired when a tile model is loaded
+     * @event OGC3DTilesLayer#load-model
+     * @type {Object}
+     * @property {THREE.Group} scene - the model (tile content) parsed in a THREE.GROUP
+     * @property {Object} tile - the tile metadata from the tileset
+     */
+    LOAD_MODEL: 'load-model',
+    /**
+     * Fired when a tile model is disposed
+     * @event OGC3DTilesLayer#dispose-model
+     * @type {Object}
+     * @property {THREE.Group} scene - the model (tile content) that is disposed
+     * @property {Object} tile - the tile metadata from the tileset
+     */
+    DISPOSE_MODEL: 'dispose-model',
+    /**
+     * Fired when a tiles visibility changes
+     * @event OGC3DTilesLayer#tile-visibility-change
+     * @type {Object}
+     * @property {THREE.Group} scene - the model (tile content) parsed in a THREE.GROUP
+     * @property {Object} tile - the tile metadata from the tileset
+     */
```

Comment:
Missing parameter in event.
```suggestion
     * @property {boolean} visible - the tile visible state
     */
```